roles/app/templates/application.yml.j2

```diff
@@ -14,3 +14,11 @@ LOCALE: {{ locale }}
 CURRENCY: {{ currency }}
 # Spree zone.
 CHECKOUT_ZONE: {{ checkout_zone }}
+
+GOOGLE_MAPS_API_KEY: {{ google_maps_api_key }}
+
+S3_ACCESS_KEY: {{ s3_access_key }}
```

Comment:
You mean different S3 accounts could be used for images and backups? Why would that be likely? Wouldnt it just be different buckets?